modules/cluster-and-ha-management/pages/set-up-crr.adoc

```diff
@@ -38,15 +32,8 @@ Use `gadmin backup` to xref:tigergraph-server:backup-and-restore:backup-cluster.
 
 For how to restore a cluster from a backup that was created from another database cluster (cross cluster), refer to xref:tigergraph-server:backup-and-restore:cross-cluster-backup.adoc[].
 
-If you are setting up both the primary cluster and the DR cluster from scratch, you can skip Steps 1, 2, and 4 and only perform Step 3.
-
 === Enable CRR on the DR cluster
 
-The command `gadmin backup restore --dr` does two things:
-
-* Enables CRR.
-* Restores the backup from the primary cluster.
-
```

Comment:
`gadmin backup restore <tag> --dr`
When both PR and DR are built from scratch, the `tag` can be empty.

Reply:
L79~L80 was already clear on the usage of this command.